### PR TITLE
[MM-61259] Fix errcheck issues in server/channels/app/import_utils_test.go

### PR DESCRIPTION
**Pull request by Gesare5**
#### Summary

Fixes errcheck issue in import_utils_test.go
Removes `channels/app/import_utils_test.go` from .golangci.yml

#### Ticket Link

Fixes  [#28950](https://github.com/mattermost/mattermost/issues/28950)
Jira  [https://mattermost.atlassian.net/browse/MM-61259](https://mattermost.atlassian.net/browse/MM-61259)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/import_utils_test.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ func TestGeneratePassword(t *testing.T) {
 
 	t.Run("Should not fail on concurrent calls", func(t *testing.T) {
 		for i := 0; i < 10; i++ {
-			go generatePassword(10)
+			_, err := generatePassword(10)
+			require.NoError(t, err)
```
Comment (hanzei) [inline server/channels/app/import_utils_test.go:39]: You removed the concurrency from the code. Can you please add it back? Thanks!

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Review by hanzei — APPROVED** [review]
Nice one

**Review by nickmisasi — APPROVED** [review]
LGTM. Thanks @Gesare5 🚀

**Pull request merged**